Client/src/Validation/validation.js

```diff
@@ -18,10 +18,19 @@ const registerValidation = joi.object({
       "string.empty": "Email is required",
     }),
 
-  password: joi.string().min(8).required().messages({
-    "string.min": "Password must be at least 8 characters",
-    "string.empty": "Password is required",
-  }),
+  password: joi
+    .string()
+    .min(8)
+    .pattern(
+      new RegExp("^(?=.*[A-Z])(?=.*\\d)(?=.*[@$!%*?&])[A-Za-z\\d@$!%*?&]{8,}$")
```

Comment:
This regex has redundancy in it, we do not need to check for 8 char length, joi is already doing that, it can be removed for clarity